README.md

```diff
@@ -6,7 +6,7 @@ Overwatch is a creation of Zac Sturgess. See also the [list of contributors](htt
 
 **Warning:** Overwatch is still in the process of being developed, and is unstable. Also, some of the code was written at 3am and needs a complete overhaul. You have been warned.
 
-![Overwatch Dashboard](app/Resources/docs/screenshots/overwatch_dashboard.png)
```

Comment:
Can you delete the old screenshot for completeness, @DesignyourCode 
Don't worry about the tour, I'll remove that in a future PR anyway.